dm/config/task.go

```diff
@@ -1175,6 +1177,7 @@ type SyncerConfigForDowngrade struct {
 	SafeModeDuration string `yaml:"safe-mode-duration,omitempty"`
 	Compact          bool   `yaml:"compact,omitempty"`
 	MultipleRows     bool   `yaml:"multipleRows,omitempty"`
+	AutoIDCacheSize  uint64 `yaml:"auto-id-cache-size,omitempty"`
```

Comment:
It's better to skip this field to `SyncerConfigForDowngrade`. This structure is used to export the config and import it to an old version of DM cluster. The unknown field will cause problems